plugins/scaffolder/src/components/secrets/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export { useSecretsContext } from './Context';
```

Comment:
wanted to export `SecretsContextProvider` too right

Reply:
Hmm this is up for debate. Would be nice to keep this private inside the plugin so you can't hook into it and grab the secrets but maybe for allowing people to write their own scaffolder page we need to?